staramr/blast/results/BlastHitPartitions.py

```diff
@@ -21,14 +21,10 @@ def append(self, hit):
         :param hit: The hit to add.
         :return: None
         """
-        if hit.get_contig_start() > hit.get_contig_end():
+        if hit.get_genome_contig_start() > hit.get_genome_contig_end():
```

Comment:
I don't think so, but I wasn't sure.  I figured it doesn't hurt to raise an `Exception`, just in case that happens. Plus, it makes it explicit what I expect from the start/end coordinates.